docs/en/Command-Line-Interface.md

```diff
@@ -176,9 +176,16 @@ For example, `cp` can be used to copy files between Under file systems.
 
 ### du
 
-The `du` command outputs the size of a file. If a directory is specified, it will output the aggregate size of all files in the directory and its children directories.
+The `du` command outputs the total size and in Alluxio size of files and folders. 
+If the Alluxio space is unexpectedly over utilized, `du` can help detect which folders are taking up the most space.
 
-For example, if the Alluxio space is unexpectedly over utilized, `du` can be used to detect which folders are taking up the most space.
+If a directory is specified, it will display the total size and in Alluxio size of all files in this directory.
+If the `-s` option is used, it will display the aggregate summary of file lengths being displayed.
+
+By default, `du` prints the size in bytes. If the `-h` option is used, it will print sizes in human readable format (e.g., 1KB 234MB 2GB).
+
+The `--header` option will print the header showing what kind of size each column belongs to.
```

Comment:
I would reverse this since new users encountering the command won't know what each column means, and they also won't know about the `--header` option. Having a header is better for interactive purposes, skipping header is more for programmatic usage. @apc999 what do you think?